README.md

```diff
@@ -121,26 +126,142 @@ packages](./docs/how_to_write_packages.md).
 
 ![Preflight package structure diagram](./docs/images/preflight_package.png)
 
-## Get Preflight
+## Installation
+
+This section outlines how to install Preflight.
+
+### Agent
+
+The following instructions walk through the installation of the Preflight agent
+to gather data about cluster pods and send them to the backend for analysis.
+
+**To complete the secret manifest below, you will need to have a Preflight
+token.**
+
+First create a namespace for the preflight components:
+
+```yaml
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: preflight
+```
+
+Next create a secret like the following, substituting your token:
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: agent-config
+  namespace: preflight
+type: Opaque
+stringData:
+  config.yaml: |
+    schedule: "* * * * *"
+    token: # enter your token here
+    endpoint:
+      protocol: https
+      host: preflight.jetstack.io
+      path: /api/v1/datareadings
+    data-gatherers:
+    - name: "pods"
+      kind: "k8s"
+      config:
+        resource-type:
+          resource: pods
+          version: v1
+```
+
+Now create a service account with permissions to read cluster resources:
+
+```yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: agent
+  namespace: preflight
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: preflight-agent-cluster-viewer
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  # will be able to view all resources, but not rbac and secrets
+  name: view
+subjects:
+- kind: ServiceAccount
+  name: agent
+  namespace: preflight
+```
+
+Finally deploy the agent:
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: agent
+  namespace: preflight
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: agent
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: agent
+    spec:
+      serviceAccountName: agent
+      volumes:
+      - name: config
+        secret:
+          secretName: agent-config
+      containers:
+      - name: agent
+        image: quay.io/jetstack/preflight:7d4fa467258b7592d68fd660f1fd1d42e7332231
+        args:
+        - "agent"
+        - "-c"
+        - "/etc/secrets/preflight/agent/config.yaml"
+        volumeMounts:
+        - name: config
+          mountPath: "/etc/secrets/preflight/agent"
+          readOnly: true
+        resources:
+          requests:
+            memory: "200Mi"
+            cpu: "200m"
+          limits:
+            memory: "200Mi"
+            cpu: "200m"
+```
+
+### Preflight V1 (check)
```

Comment:
Maybe to make things cleaner all the info about check should be moved out to another file in `docs/` and just mentioned and linked once above. Might also be best for another PR.

Reply:
I can move them out if that's the popular vote.

My reasoning here was that we have documentation in a lot of places - this is becoming harder to keep in sync as we deprecate check. 

Until then, I wanted to keep the changes minimal and quite self contained until we're ready to remove check references completely...

Reply:
Yeah I can see the reasoning there. I think I just don't like the cluttered feel in the mean time but I can always just get over it.

My thinking was that we actually move all check specific stuff into `docs/check.md`, refer to this once in `README.md` and otherwise everything can focus on the agent and new SaaS way of working.

Reply:
Sounds reasonable, I'll see what that looks like.

Reply:
ffe6c83351954fec31df77d8c638f3584b4fbcb7 is an attempt at this.

Reply:
I've tried to move all the check docs and examples to one place - leaving the new readme as a clear process for installing the agent.